src/content/documentation/publish/self-distribution.md

```diff
@@ -50,7 +50,7 @@ For more information on how to submit an add-on for distribution on AMO or self-
 
 When you choose to distribute an add-on yourself, they can be installed using the following methods:
 
-- [Web download](#web-download)—make your extension available on a suitable web accessible server and when the user downloads the signed add-on file Firefox installs it.
+- [Web download](#web-download)—make your extension available on a suitable web accessible server and when the user downloads the signed add-on file Firefox installs it. The signed add-on file must have a `Content-Type: application/x-xpinstall` HTTP response header.
```

Comment:
Nit: The web accessible server must serve the signed addon xpi file with `Content-Type: application/x-xpinstall` in the HTTP response headers.

Reply:
Updated using reworded suggestion